soccer/FieldView.hpp

```diff
@@ -41,6 +41,7 @@ class FieldView : public QWidget {
 
     void rotate(int value);
 
+    const Geometry2d::TransformMatrix& getTeamToWorld() { return _teamToWorld; }
```

Comment:
Let's put a `const` in between `()` and `{` to indicate that calling the method doesn't modify `FieldView`.